packages/caver-utils/src/utils.js

```diff
@@ -629,19 +669,19 @@ function parsePrivateKey(privateKey) {
         return {
             privateKey: `0x${privateKey}`,
             address: '',
-            isHumanReadable: false,
+            type: '',
         }
     }
 
     if (!isKlaytnWalletKey(privateKey)) throw new Error(`Invalid KlaytnWalletKey format.`)
 
-    const humanReadableFlag = privateKey.slice(66, 68)
-    if (humanReadableFlag === '01') throw new Error('HumanReadableAddress is not supported yet.')
+    const type = privateKey.slice(66, 68)
+    if (type === '01') throw new Error('Invalid type: Currently only type `0x00` is supported.')
```

Comment:
```suggestion
    if (type !== '00') throw new Error('Invalid type: Currently only type `0x00` is supported.')
```
How about check condition if type has not "0x00"?